### PR TITLE
Fix/batch

### PR DESCRIPTION
**Pull request by CrepeGoat**
Some fixes to make to batch. @shanlyMIDE anything else to add here?

### CODE REVIEW

**Comment by shanlyMIDE** [comment]
@CrepeGoat will this solve the test shown in this colab where I compare directly loading and integrating the data to the batch output?
https://colab.research.google.com/drive/1VOlWL_0Z6IS6_A-5lwSB5U-u-4yvP3aR#scrollTo=vA2CWXXg9hoM

**Review by CFlaniganMide — COMMENTED** [review]
Changes look fine, but was this getting caught by tests before the change was made?

**Comment by CrepeGoat** [comment]
> was this getting caught by tests before the change was made?

Great question/point: **no, it wasn't!** I'm looking to write a test to catch that discrepancy. (In fact until that's done I think I'll make this a draft PR again.)

File: endaq/batch/core.py [hunk 1]
```diff
@@ -247,6 +247,13 @@ def __init__(
         :param accel_end_margin: the numper of samples after which to reject
             recording data; cannot be used in conjunction with
             `accel_end_time`
+        :param accel_integral_tukey_percent: the alpha parameter of a tukey
```
Comment (StokesMIDE) [inline endaq/batch/core.py:250]: nbd, but Tukey should be capitalized in the docstring (where it isn't referring to a variable/parameter/function name).

**Review by StokesMIDE — APPROVED** [review]

File: endaq/batch/core.py (hunk 1 above)
Comment (CrepeGoat) [reply endaq/batch/core.py:250]: fixed in b01ea5c96503ca46794748bf9b4dba3a5ec9827c. thanks!

**Pull request merged**